_posts/2016-04-15-our-hiring-process.markdown

```diff
@@ -24,15 +24,22 @@ I realize that there are great developers that have never made any open source c
 
 We want to see some recent code samples and evaluate you based on your work, not on your work history. **Experience** is important but **passion** is more important.
 
-## First Interview
+## Non Technical Interview
 
 If we like what we see, we will send you an email to coordinate an initial interview.
 
-In this interview we will talk about your experience, what you know, what you are passionate about, what you are looking for, what we are looking for, and some concepts that are really important to us.
+In this interview we will talk about your experience, what you are passionate about, what you are looking for, what we are looking for, and some concepts that are really important to us.
 
 In this stage we want to get a sense of what you are looking for and if we are a **good fit for you**.
+So is it time for you ask all your questions about the company and the position that you are applying for.
```

Comment:
@cleicar This sounds a little weird. Maybe like this it would sound a little clearer: 

```
Now is a good time to ask all your questions about the company and the position you're interviewing for. 
```